contrib/omhiredis/omhiredis.c

```diff
@@ -185,7 +194,9 @@ static rsRetVal writeHiredis(uchar* key, uchar *message, wrkrInstanceData_t *pWr
 	 * happened, in which case abort. otherwise
 	 * increase our current pipeline count
 	 * by 1 and continue. */
-	int rc;
+	int rc, expire;
+	size_t msgLen;
+	char *formattedMsg = NULL;
```

Comment:
The compiler is right: if this line is not reached, the variable exists, is used during cleanup, but is undefined.

Suggestion: move up to top of function.